### DISCUSSION TITLE
TE3 on Windows ARM and connections to Fabric / Microsoft Synapse

### DISCUSSION THREAD

**Discussion by redstor-albe** [TabularEditor/TabularEditor3#1213] [🙋♀️TE3 Q&A]
Has anyone tried a similar configuration?  I am running a Windows 11 ARM installation, the app works fine except for connections where I need to use MSOLEDBSQL connections to Synapse.  It seems that even if the MSOLEDB installs has been done, TE3 cannot make use of it.  My next step would be to install the same on a X86 based Windows and then compare the files in Program Files vs Program Files (X86) vs Program Files (ARM).

**Comment by otykier** [comment]
Apologies for the late reply. Did you manage to find a solution for this? In my experience, using the x64 version of both Tabular Editor and all relevant drivers, should work fine - even on an ARM machine.
